tests/Geocoder/Tests/Model/AddressFactoryTest.php

```diff
@@ -34,9 +34,9 @@ public function testCreateFromArray()
         foreach ($addresses as $address) {
             $this->assertInstanceOf('Geocoder\Model\Address', $address);
             $this->assertNull($address->getCoordinates());
-            $this->assertInstanceOf('Geocoder\Model\County', $address->getCounty());
+            $this->assertInstanceOf('Geocoder\Model\County', $address->getAdmins()[1]);
```

Comment:
I think the assertion is wrong, it should assert than it's an instance of `Geocoder\Model\Admin` (and `AdminLevel` after you change it) ;)